src/store/modules/auth.module.ts

```diff
@@ -1,17 +1,31 @@
+import { AuthTokens } from '@/api/modules/auth'
 import { createStore } from '@/helpers'
 
 interface AuthState {
-  jwt: string
+  tokens: {
+    orgId: string
+    groupId: string
+    accessToken: string
+    refreshToken: string
+  }[]
 }
 
 const [authStore, useAuthState] = createStore(
   'auth',
   {
-    jwt: '',
+    tokens: [],
   } as AuthState,
   state => ({
-    setJwt: (jwt: string) => {
-      state.jwt = jwt
+    addToken: (tokens: AuthTokens, orgId: string, groupId: string) => {
```

Comment:
addTokens, not token?

Reply:
could we define:

app tokens - is a "token"
"authTokens" - is an group of access + refresh tokens?